Odata-docs/client/v7/get-response-content.md

```diff
@@ -32,3 +39,34 @@ static void Main(string[] args)
     }
 }
 ```
+The `DataServiceResponse` has the following properties that enable you to access additional information about a modification response:
+
+- [BatchHeaders](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchheaders) - contains headers from an HTTP response associated with a batch request.
+- [BatchStatusCode](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchstatuscode)- contains status code from an HTTP response associated with a batch request.
+- [IsBatchResponse](/dotnet/api/microsoft.odata.client.dataserviceresponse.isbatchresponse) - gets a Boolean value that indicates whether the response contains multiple results.
+
+The `DataServiceResponse` is enumerated to retrieve responses to operations being tracked by [OperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) objects within the `DataServiceResponse`.
+
+## Query Responses
+We get a `Data Query Response` after calling `Execute()` or `ExecuteAsync()` on `DataServiceQueries`.
```

Comment:
Is `Data Query Response` the same thing as `QueryOperationResponse`?

Reply:
`QueryOperationResponse` is the technical term. 
`Data Query Response` is the verbose term, meaning that `QueryOperationResponse` is a response from a Query request.

Reply:
I have changed to `QueryOperationResponse` to eliminate confusion.

Reply:
#Resolved